lib/Database/Database.cpp

```diff
@@ -105,12 +115,14 @@ Database::Implementation::create(StringRef path, bool readonly, Optional<size_t>
 
   SmallString<128> savedPathBuf = versionPath;
   llvm::sys::path::append(savedPathBuf, "saved");
-  SmallString<128> processPathBuf = versionPath;
+  SmallString<128> prefixPathBuf = versionPath;
 #if defined(WIN32)
-  llvm::raw_svector_ostream(processPathBuf) << "/p" << GetCurrentProcess();
+  llvm::raw_svector_ostream(prefixPathBuf) << "/p" << GetCurrentProcess();
 #else
-  llvm::raw_svector_ostream(processPathBuf) << "/p" << getpid();
+  llvm::raw_svector_ostream(prefixPathBuf) << "/p" << getpid();
 #endif
+  llvm::raw_svector_ostream(prefixPathBuf) << "-";
```

Comment:
Is this needed? It looks like createUniqueDirectory adds its own hyphen.

Reply:
I noticed but I wanted to ensure a hyphen exists (it is important in other part of the code) and I didn't want to depend on the underlying implementation.